ntoskrnl/ke/amd64/cpu.c

```diff
@@ -117,7 +153,7 @@ KiGetFeatureBits(VOID)
     CPU_INFO CpuInfo;
 
     /* Get the Vendor ID */
-    Vendor = KiGetCpuVendor();
+    Vendor = Prcb->CpuVendor;
```

Comment:
Here, it is already initialized?

Reply:
Yes, here it is. I think I'm going to add a comment on the first call.